pycfmodel/model/resources/iam_user.py

```diff
@@ -28,13 +28,12 @@ def __init__(self, logical_id, value):
 
         self.user_name = None
         self.path = None
-        self.policies = []
 
         self.policies = self.get_policies(
-            value.get("Properties", {}).get("Policies"),
+            value.get("Properties", []).get("Policies"),
```

Comment:
Hey! Maybe I'm wrong, but why did you change this from a dict to a list? Btw, a list doesn't have a `get` method.

Reply:
Oh my, because I was not paying attention. My intent was to have the default of getting `value['Properties']['Policies']` equal and empty list if there are no polices or the `Policies` key is not present. Previously there were a handful of issues where parts of the code were expecting lists  and received strings or other. I will update the code. Sorry let me know if you see any other issues.